core/Pos/Binary/Class.hs

```diff
@@ -58,7 +58,10 @@ module Pos.Binary.Class
        , biSize
        ) where
 
+import           Universum
```

Comment:
Yes, good catch. I want to add such rule, like _If you're using custom prelude, put it before other imports in separate section_. This makes sense to me. But such `style-guide` changes should be discussed. So until now I just moved imports at top while nobody noticed. Now, if nobody  mind, I can upgrade `style-guide`.